tests/cursor_tests.rs

```diff
@@ -219,6 +219,45 @@ pub async fn cursor_pagination(db: &DatabaseConnection) -> Result<(), DbErr> {
         [Row { id: 6 }, Row { id: 7 }]
     );
 
+    #[derive(DerivePartialModel, FromQueryResult, Debug, PartialEq)]
+    #[sea_orm(entity = "Entity")]
+    struct PartialRow {
+        #[sea_orm(from_col = "id")]
+        id: i32,
+        #[sea_orm(from_expr = "sea_query::Expr::col(Column::Id).add(1000)")]
+        id_shifted: i32,
+    }
+
+    let mut cursor = cursor.into_partial_model::<PartialRow>();
+
+    assert_eq!(
+        cursor.first(2).all(db).await?,
+        [
+            PartialRow {
+                id: 6,
+                id_shifted: 1006,
+            },
+            PartialRow {
+                id: 7,
+                id_shifted: 1007,
+            }
+        ]
+    );
+
+    assert_eq!(
+        cursor.first(3).all(db).await?,
+        [
+            PartialRow {
+                id: 6,
+                id_shifted: 1006,
+            },
+            PartialRow {
+                id: 7,
+                id_shifted: 1007,
+            }
+        ]
+    );
+
     // Fetch JSON value
 
     let mut cursor = cursor.into_json();
```

Comment:
After "into_partial_model",  the result has a
 extra field "id_shifted",  but the following test case do not update assertion. That cause ci failure.

Reply:
Correct! I just fixed it https://github.com/SeaQL/sea-orm/pull/1597/commits/c21d82d67819fe4c79218364ec9c3bd23f5bc904